src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -110,13 +110,6 @@
        :accessory           (when mnemonic
                               [components.common/counter {:size 22} 1])
        :on-press            #(re-frame/dispatch [:navigate-to :privacy-and-security])}]
-     (when config/quo-preview-enabled?
```

Comment:
Removing quo preview entry from profile. We don't use it anymore and the app just crashes on pressing it.